srcpkgs/ccsm/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'ccsm'
 pkgname=ccsm
 version=0.8.16
-revision=2
+revision=3
 noarch=yes
 build_style=python3-module
 hostmakedepends="automake intltool libtool pkg-config python3"
 makedepends="compiz-core-devel compizconfig-python python3-cairo-devel pygtk-devel"
-depends="compizconfig-python python3-cairo"
+depends="compizconfig-python python3-cairo python3-gobject"
```

Comment:
keep the depends back in the $CROSS_BUILD guard, just change it there.

This change will break cross builds.

Reply:
ok, will do.